packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -66,7 +71,11 @@ export const ParticipantListWithHeading = (props: {
   return (
     <Stack className={participantListStack}>
       <Stack.Item styles={subheadingStyleThemed} aria-label={title}>
-        {title}
+        {title}{' '}
```

Comment:
can we string.format this instead of having a bunch of strings that happen to fit next to eachother?